### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by jeetbhalani15**
image
this is a temporary preview layout

### CODE REVIEW

File: components/Badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+@import "/reset.css";
+.box{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```
Comment (rohanmathur91) [inline components/Badges/badges.css:5]: Have a utility class for flex, so that you don't repeat it.

File: components/cards/card.css [hunk 2]
```diff
@@ -0,0 +1,241 @@
+
+@import "/reset.css";
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
```
Comment (rohanmathur91) [inline components/cards/card.css]: You have reset.css so why resetting it again?

**Review by rohanmathur91 — COMMENTED** [review]
Added a few comments.

File: components/Badges/badges.css (hunk 1 above)
Comment (jeetbhalani15) [reply components/Badges/badges.css:5]: ok done

File: components/Alert/alert.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
+
+@import "/reset.css";
+.alert-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (aniketsingh13) [inline components/Alert/alert.css:7]: if these code repeat you can put it in the utility class

**Review by aniketsingh13 — COMMENTED** [review]
consider the changes other wise good to go

File: components/Alert/alert.css (hunk 3 above)
Comment (jeetbhalani15) [reply components/Alert/alert.css:7]: yes done

**Review by aniketsingh13 — APPROVED** [review]

**Review by Jay-Gadhiya — APPROVED** [review]

**Pull request merged**